src/tests/flyer.test.ts

```diff
@@ -191,16 +191,57 @@ describe('incrementShoutouts tests', () => {
 
 describe('getTrending tests', () => {
   test('getTrendingFlyers - get 5 trending flyers', async () => {
-    const trendingFlyers = await FlyerFactory.createSpecific(5, {
-      isTrending: true,
-    });
-    const notTrendingFlyers = await FlyerFactory.createSpecific(5, {
-      isTrending: false,
+    // Shuffle order of trendiness
+    const randomTrendiness = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9].sort(() => Math.random() - 0.5);
+    // Create flyers with random trendiness
+    const randomFlyers = await FlyerFactory.createSpecificsByIndex(
+      randomTrendiness.length,
+      (index) => {
+        return {
+          trendiness: randomTrendiness[index],
+          title: `title${randomTrendiness[index]}`,
+        };
+      },
+    );
```

Comment:
This approach is interesting, but I don't see any major advantage to this call over
```js
const randomFlyers = await FlyerFactory.create(n);
for (let i = 0; i < randomFlyers.length; i++) {
  randomFlyers[i].trendiness = randomTrendiness[index]
  randomFlyers[i].title = `title${randomTrendiness[index]}`;
  // insert any other attributes you want to modify here
}
```

The purpose of writing static methods like this is to re-use code and reduce complexity but in this case the for loop approach takes essentially the same amount of lines (less when you count lines saved by not writing the function).

Even if we write a for loop every time we want to modify the attributes of the random flyers in a test case, it'll take essentially the same amount of work as calling this function. We could technically pull out your anonymous function and try to re-use that but I think this just convolutes the code too much.

I tend to really prefer simplicity of code above all else, so I think we should just go with the straight for loop approach here

Reply:
I agree - honestly I didn't know that you could just write a for loop, and I was inspired by Kotlin's `forEachIndexed` style of functional programming. But I see how that adds complexity for almost no gain in efficiency, so a for loop works better in this case. Thanks for the tip and I removed the function from the code.